code/game/antagonist/station/cultist.dm

```diff
@@ -22,7 +22,7 @@
 /decl/special_role/cultist
 	name = "Cultist"
 	name_plural = "Cultists"
-	blacklisted_jobs = list(/datum/job/ai, /datum/job/cyborg, /datum/job/submap)
```

Comment:
maybe whole no-synth thing should be a flag rather than having to add it to every list like this.

Reply:
Made it use event categories instead.